dataland-frontend/tests/e2e/utils/LksgUpload.ts

```diff
@@ -194,6 +194,65 @@ function fillRequiredLksgFieldsWithDummyData(): void {
   cy.get("select[name=totalRevenueCurrency]").select("EUR");
 }
 
+/**
+ * Adds a new product and fills its form
+ * @param productName the name of the product to fill in
+ * @param productionSteps the production steps to fill in
+ * @param description the description to fill in
+ */
+function fillNewProduct(productName: string, productionSteps: string[], description?: string): void {
+  cy.get('button[data-test="addNewProductButton"]').click();
+  const productFormElementSelector = '[data-test="productSection"] [data-test="productFormElement"]';
+  cy.get(productFormElementSelector).last().find('input[name="productName"]').type(productName);
+  if (productionSteps.length > 0 && productionSteps.some((productionStep) => productionStep.length > 0)) {
+    cy.get(productFormElementSelector)
+      .last()
+      .find('input[data-test="listOfElementsInput"]')
+      .type(productionSteps.join(", "));
+    cy.get(productFormElementSelector).last().find('[data-test="addProductionStep"]').click();
+  }
+  if (description) {
+    cy.get(productFormElementSelector).last().find('textarea[name="relatedCorporateSupplyChain"]').type(description);
+  }
+}
+
+/**
+ * Adds some products to the most important products form
+ */
+function fillInMostImportantProducts(): void {
+  fillNewProduct("Test Product 1", ["first", "second"], "Description of something");
+  fillNewProduct("Test Product 2", []);
+}
+
+/**
+ * Fills out Procurement Categories
+ */
+export function fillInProcurementCategories(): void {
+  cy.get('[data-test="dataPointToggleButton"]').first().click();
+  cy.get('[data-test="suppliersPerCountryCode"] .p-multiselect').should("exist").click();
+
+  const selectedCountries: string[] = [];
+
+  cy.get(".p-multiselect-panel ul.p-multiselect-items li.p-multiselect-item")
+    .should("exist")
+    .each(($item, index) => {
+      if (index < 2) {
+        const coutryName = $item[0].innerText.split(" ").slice(0, -1).join(" ");
+        cy.wrap($item).click();
+        selectedCountries.push(coutryName);
+      }
+    });
+
+  cy.get('[data-test="directSuppliersHeader"]').should("exist").should("contain", "Number of Direct Supliers");
```

Comment:
typo: "Suppliers"

Reply:
Done.

Reply:
In the Frontend it still has a typo:

![image](https://github.com/d-fine/Dataland/assets/99255296/3eb42d7d-c85c-4f61-b115-9011ce4fcca5)


Also see my comment to use `DataModelsTranslations.ts` instead of hardcoding the string for the label there.

Reply:
👍